modules/cluster-autoscaler-cr.adoc

```diff
@@ -50,13 +51,23 @@ spec:
 <7> Optional: Specify the type of GPU node to deploy. Only `nvidia.com/gpu` and `amd.com/gpu` are valid types.
 <8> Specify the minimum number of GPUs to deploy in the cluster.
 <9> Specify the maximum number of GPUs to deploy in the cluster.
-<10> In this section, you can specify the period to wait for each action by using any valid link:https://golang.org/pkg/time/#ParseDuration[ParseDuration] interval, including `ns`, `us`, `ms`, `s`, `m`, and `h`.
-<11> Specify whether the cluster autoscaler can remove unnecessary nodes.
-<12> Optional: Specify the period to wait before deleting a node after a node has recently been _added_. If you do not specify a value, the default value of `10m` is used.
-<13> Optional: Specify the period to wait before deleting a node after a node has recently been _deleted_. If you do not specify a value, the default value of `0s` is used.
-<14> Optional: Specify the period to wait before deleting a node after a scale down failure occurred. If you do not specify a value, the default value of `3m` is used.
-<15> Optional: Specify the period before an unnecessary node is eligible for deletion. If you do not specify a value, the default value of `10m` is used.
-<16> Optional: Specify the _node utilization level_ below which an unnecessary node is eligible for deletion. The node utilization level is the sum of the requested resources divided by the allocated resources for the node, and must be a value greater than `"0"` but less than `"1"`. If you do not specify a value, the cluster autoscaler uses a default value of `"0.5"`, which corresponds to 50% utilization. This value must be expressed as a string.
+<10> Specify the logging verbosity level between `0` and `10`. The log levels correspond to the following information being shown: 
++
+--
+* `0` - `1`: silent
+* `2` - `5`: all flags
+* `6`: API requests
+* `7`: response headers
+* `8` - `10`: response body
```

Comment:
Should/could these be more granular (like, is there a difference for every integer value as they step from 0 through 10)?

Reply:
i think this is a good level of granularity, imo i would make less granular but i'm not entirely sure about some of the HTTP logging levels.

i looked through the source code for the autoscaler and it's really all over the place. we have kept our cloud-provider (clusterapi) consistent at level `4`, but the main code ranges from `0` to `5` in places. those higher values `7+` are exposing things in the HTTP client, so i'm not entirely sure.

this is my general advice when working with users and debugging;
* `1`, default level
* `4`, if you want to see more details about autoscaler is doing, especially the clusterapi provider
* `9`, _firehose_ in case you think there is a problem happening with the HTTP calls to the kubernetes API server and need to debug at the protocol level.